cmd/syncthing/main.go

```diff
@@ -143,218 +111,211 @@ Debugging Facilities
+	DeviceID         bool   `help:"Show the device ID"`
+	GenerateDir      string `name:"generate" placeholder:"PATH" help:"Generate key and config in specified dir, then exit"`
+	GUIAddress       string `name:"gui-address" placeholder:"URL" help:"Override GUI address (e.g. \"http://192.0.2.42:8443\")"`
+	GUIAPIKey        string `name:"gui-apikey" placeholder:"API-KEY" help:"Override GUI API key"`
+	HideConsole      bool   `help:"Hide console window (Windows only)"`
+	HomeDir          string `name:"home" placeholder:"PATH" help:"Set configuration and data directory"`
+	LogFile          string `placeholder:"PATH" help:"Log file name (see below)"`
+	LogFlags         int    `placeholder:"BITS" help:"Select information in log line prefix (see below)"`
+	LogMaxFiles      int    `placeholder:"N" name:"log-max-old-files" help:"Number of old files to keep (zero to keep only current)"`
+	LogMaxSize       int    `placeholder:"BYTES" help:"Maximum size of any file (zero to disable log rotation)"`
+	NoBrowser        bool   `help:"Do not start browser"`
+	NoRestart        bool   `env:"STNORESTART" help:"Do not restart Syncthing when exiting due to API/GUI command, upgrade, or crash"`
+	NoDefaultFolder  bool   `env:"STNODEFAULTFOLDER" help:"Don't create the \"default\" folder on first startup"`
+	NoUpgrade        bool   `env:"STNOUPGRADE" help:"Disable automatic upgrades"`
+	Paths            bool   `help:"Show configuration paths"`
+	Paused           bool   `help:"Start with all devices and folders paused"`
+	Unpaused         bool   `help:"Start with all devices and folders unpaused"`
+	Upgrade          bool   `help:"Perform upgrade"`
+	UpgradeCheck     bool   `help:"Check for available upgrade"`
+	UpgradeTo        string `placeholder:"URL" help:"Force upgrade directly from specified URL"`
+	Verbose          bool   `help:"Print verbose log output"`
+	Version          bool   `help:"Show version"`
+
+	// Debug options below
+	DebugBlockProfiler        bool          `env:"STBLOCKPROFILE" help:"Write block profiles to block-$pid-$timestamp.pprof every 20 seconds"`
+	DebugCPUProfile           bool          `help:"Write a CPU profile to cpu-$pid.pprof on exit" env:"CPUPROFILE"`
+	DebugDBIndirectGCInterval time.Duration `env:"STGCINDIRECTEVERY" help:"Database indirection GC interval"`
+	DebugDBRecheckInterval    time.Duration `env:"STRECHECKDBEVERY" help:"Database metadata recalculation interval"`
+	DebugDeadlockTimeout      int           `placeholder:"SECONDS" env:"STDEADLOCKTIMEOUT" help:"Used for debugging internal deadlocks"`
+	DebugGUIAssetsDir         string        `placeholder:"PATH" help:"Directory to load GUI assets from" env:"STGUIASSETS"`
+	DebugHeapProfile          bool          `env:"STHEAPPROFILE" help:"Write heap profiles to heap-$pid-$timestamp.pprof each time heap usage increases"`
+	DebugNetworkProfilerURL   string        `placeholder:"URL" env:"STPROFILER" help:"Network profiler listen URL"`
```

Comment:
I'd rename the various profilers to `DebugProfile...`, to group them under their common denominator.